src/js/components/StarRating/StarRating.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import { Star, StarOutline } from 'grommet-icons';
+import { StarRatingPropTypes } from './propTypes';
+import { RadioButtonGroup } from '../RadioButtonGroup';
+
+const StarRating = ({ color, scale = 5, ...rest }) => {
```

Comment:
Do we need the `scale` property for the first version? Just checking.

Reply:
not necessarily